### PR TITLE
feat: Added my-activity-section [#1452]

### PR DESCRIPTION
**Pull request by anujnema5**
## Description
#1452

Added Feedback, fixed UI styling integrated some utility functions integrated canny API, fetched user-specific data 

## What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build

## Have you tested locally?

- [ ] 👍 yes
- [ ] 🙅 no, because I am lazy

## Added tests?

- [ ] 👍 yes
- [ ] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md

### CODE REVIEW

**Review by gc-codes — APPROVED** [review]

File: .vscode/settings.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
   },
   "svelte.enable-ts-plugin": true,
   "[typescript]": {
-    "editor.defaultFormatter": "esbenp.prettier-vscode"
+    "editor.defaultFormatter": "vscode.typescript-language-features"
   },
   "[typescriptreact]": {
```
Comment (Abhishek9503) [inline .vscode/settings.json:10]: Rever this

File: src/packages/@support/features/feedback-section/layout/FeedbackPost.svelte [hunk 2]
```diff
@@ -73,11 +76,19 @@
...
+      post = await res?.data;
+      console.log(post);
+      console.log(post, "post");
+      createdAt = timeAgo(post?.created);
+      postImages = post?.imageURLs;
+      comments = await fetchComments(id);
+    } catch (error) {
+      loading = false;
+    }
   });
```
[the remark this answers is not in the document]
Comment (anujnema5) [reply src/packages/@support/features/feedback-section/layout/FeedbackPost.svelte]: Sure will remove console but if API call gets failed in any condition then loading should become false

File: .vscode/settings.json (hunk 1 above)
Comment (anujnema5) [reply .vscode/settings.json:10]: okay

File: src/packages/@library/ui/spinner/Spinner.svelte [hunk 3]
```diff
@@ -28,4 +28,9 @@
       transform: rotate(360deg);
     }
   }
+
+  .small {
+    width: 50px;
+    height: 50px;
+  }
 </style>
```
Comment (itsmdasifraza) [inline src/packages/@library/ui/spinner/Spinner.svelte:36]: not required for now

File: src/packages/@library/ui/like/Like.svelte [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+    x="0"
+    y="0"
+    width="18"
+    height="18"
+  >
+    <rect width="18" height="18" fill="#D9D9D9" />
+  </mask>
+  <g mask="url(#mask0_11537_5384)">
+    <path
+      d="M13.5 15.75H5.25V6L10.5 0.75L11.4375 1.6875C11.525 1.775 11.5969 1.89375 11.6531 2.04375C11.7094 2.19375 11.7375 2.3375 11.7375 2.475V2.7375L10.9125 6H15.75C16.15 6 16.5 6.15 16.8 6.45C17.1 6.75 17.25 7.1 17.25 7.5V9C17.25 9.0875 17.2375 9.18125 17.2125 9.28125C17.1875 9.38125 17.1625 9.475 17.1375 9.5625L14.8875 14.85C14.775 15.1 14.5875 15.3125 14.325 15.4875C14.0625 15.6625 13.7875 15.75 13.5 15.75ZM6.75 14.25H13.5L15.75 9V7.5H9L10.0125 3.375L6.75 6.6375V14.25ZM5.25 6V7.5H3V14.25H5.25V15.75H1.5V6H5.25Z"
```
Comment (itsmdasifraza) [inline src/packages/@library/ui/like/Like.svelte:29]: this can be shifted to icons directory

File: src/packages/@library/ui/like/index.ts [hunk 5]
```diff
@@ -0,0 +1 @@
+export { default as Like } from './Like.svelte'
```
Comment (itsmdasifraza) [inline src/packages/@library/ui/like/index.ts:1]: this can also be shifted to icons directory

File: src/packages/@app/services/canny.service.ts [hunk 6]
```diff
@@ -120,35 +121,64 @@ export class CannyIoService {
...
+        status,
+        search
+      },
+      headers: {
+        "Content-type": ContentTypeEnum["application/x-www-form-urlencoded"],
+      },
+    });
+    console.log(response)
+    return response
+  }
```
Comment (itsmdasifraza) [inline src/packages/@app/services/canny.service.ts:139]: this can be handled using existing function

**Review by itsmdasifraza — APPROVED** [review]

**Pull request merged**